src/data/set/finite.lean

```diff
@@ -431,6 +431,19 @@ let ⟨I, Ifin, hI⟩ := finite_subset_Union tfin h in
         exact H }
     end⟩
 
+/-- Union distributes of finite directed intersection. -/
```

Comment:
I don't get this docstring. There is no `directed` assumption anywhere; the intersection is not directed of monotone, but the union is.

Reply:
Fixed